applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -274,7 +274,6 @@ void MmgProcess<TMMGLibrary>::ExecuteFinalize()
 
     // In case of considering meric tensor
     if (it_node_begin->Has(r_tensor_variable)) {
-        #pragma omp parallel for
```

Comment:
maybe add a comment that those loops cannot be done in parallel?

Reply:
Or even better adding a test